perllib/FixMyStreet/Cobrand/TfL.pm

```diff
@@ -30,4 +30,23 @@ sub enter_postcode_text {
     return 'Enter a London postcode, or street name and area';
 }
 
+sub owns_problem {
```

Comment:
I think this would be clearer overall if `is_council` is set to 0, which fixes a few other places that would then assume it could call TfL a council etc, and then this function is never called and isn't necessary, as far as I can see. Looks like index-steps.html might be only thing needing to then be added.